ui/src/components/Tasks/AddSample.jsx

```diff
@@ -9,7 +9,7 @@ import { useForm } from 'react-hook-form';
 import { hideTaskParametersForm } from '../../actions/taskForm';
 
 const REQUIRED_MSG = 'This field is required';
-const PATTERN = /^[\w+:-]*$/u;
+const PATTERN = /^[\w+\-:]*$/u;
```

Comment:
There's now a regex linting plugin. It enforces a lot of things, including that characters inside `[]` be sorted in a specific order for consistency.

Reply:
Wow ;)